dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -53,6 +53,13 @@ service AppCallbackHealthCheck {
   rpc HealthCheck(google.protobuf.Empty) returns (HealthCheckResponse) {}
 }
 
+// AppCallbackBulkSubscribe V1 is an optional extension to AppCallback V1 to opt
+// for Bulk Subscription.
+service AppCallbackBulkSubscribe {
```

Comment:
I don't suggest to add new service.

As described here: https://github.com/dapr/dapr/issues/2218#issuecomment-1257627582